Tests/WhisperKitTests/UnitTests.swift

```diff
@@ -31,6 +31,46 @@ final class UnitTests: XCTestCase {
             "Failed to init WhisperKit"
         )
     }
+    func testModelIsInstalled() async throws {
+        XCTAssertTrue(
+            WhisperKit.modelInstalled(variant: "openai_whisper-tiny"),
+            "Model was not installed"
+            )
+        XCTAssertFalse(
+            WhisperKit.modelInstalled(variant: "THIS_MODEL_DOES_NOT_EXIST"),
+            "Model does not exist, but returned true"
+        )
+        XCTAssertTrue(
+            WhisperKit.modelInstalled(variant: "tiny"),
+            "Model was not installed"
+            )
+        XCTAssertTrue(
+            WhisperKit.modelInstalled(variant: "tiny.en"),
```

Comment:
You'll need some method to download this one ahead of time before checking it is installed. https://github.com/argmaxinc/WhisperKit/blob/dd2eb733e9d151379092d47886a69cc07ac6d9f0/.github/workflows/unit-tests.yml#L60 you can do that by either adding a line `make download-model MODEL=tiny.en` here to download it, or just removing this tiny.en check and only running this test for tiny.